cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split2/TextToSpeechSSMLGeneratorSuite.scala

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.azure.synapse.ml.cognitive.split2
+
+import com.microsoft.azure.synapse.ml.cognitive._
+import com.microsoft.azure.synapse.ml.cognitive.TextToSpeechSSMLGenerator
+import com.microsoft.azure.synapse.ml.core.test.fuzzing.{TestObject, TransformerFuzzing}
+import org.apache.spark.ml.util.MLReadable
+import org.apache.spark.sql.DataFrame
+import spray.json._
+import com.microsoft.azure.synapse.ml.cognitive.TextToSpeechSSMLFormat.TextToSpeechSSMLResponseFormat
+
+class TextToSpeechSSMLGeneratorSuite extends TransformerFuzzing[TextToSpeechSSMLGenerator] with CognitiveKey  {
+
+  import spark.implicits._
+
+  def ssmlGenerator: TextToSpeechSSMLGenerator = new TextToSpeechSSMLGenerator()
+    .setUrl("https://eastus.tts.speech.microsoft.com/cognitiveservices/v1")
+    .setSubscriptionKey(cognitiveKey)
+    .setTextCol("textColName")
+    .setOutputCol("outputColName")
+    .setSSMLOutputCol("SSMLColName")
```

Comment:
1: Forgot if these cols already have defaults, but take a look at some other transformers with HasOutputCol for example
2: Not sure what the diff is between OutputCol and SSMLOutputCol, this estimator should just have OutputCol, and if an intermediate column is needed internally it should be dropped before the dataframe is returned to keep it an implementation detail

Reply:
Addressed in most recent commit